.changeset/strong-bags-type.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-css": patch
+---
+
+Disable zoom on double tap of button.
```

Comment:
This changeset should be duplicated for `@itwin/itwinui-react`. Also it should be reworded to more clearly the describe *what* is impacted and *how*.

Reply:
Added changeset for React and updated the description.

Reply:
CSS classes should not be mentioned in the React changeset.

Also, could you word these in a way that frontloads the change, and *then* explains the reason?

Reply:
That's a good point, updated to use your suggestion.

Was going to add "buttons and tabs" but tabs are `<button>`s, so left it as is.